scripts/dockerbot/pytestbase/setup.sh

```diff
@@ -116,9 +119,7 @@ git config --global user.email "me@dockerbot"
 git config --global user.name "DockerBot"
 
 # Get the GPG-Keys
-# It's not a mistake to run this twice ;)
-do_or_die "/get_keys.py >> $ARTIFACTOR_DIR/setup.txt 2>&1"
-do_or_die "/get_keys.py >> $ARTIFACTOR_DIR/setup.txt 2>&1"
+do_or_die "/get_keys.py >> $ARTIFACTOR_DIR/setup.txt 2>&1" 1
```

Comment:
We are passing `1` as `max_retry` here instead of the `sleep_duration` -> we get a single run with no sleep.